Systems/c172p-engine.xml

```diff
@@ -207,7 +207,7 @@
 
         <fcs_function name="auto-engine-mixture">
             <function> <!-- mixture to 1 at sea level in autostart, state autostarts and tutorials, default mixture setting is .95 -->
-                <sum
+                <sum>
```

Comment:
Sorry, one more thing (unless I don't understand).
The default setting for mixture in FG is 1 (not 0.95). I think that 0.95 came from onox in Autostart. See commit 3662ef5d, c172p.nas, line 14. Possibly to allow autostart at a slightly higher altitude?

Reply:
OK, so you think we also need to set the default at 1 VS .95.

Reply:
@dany93 I just tested this and it is starting with a mixture of 0 by default. The autostart is adjusting the mixture according to the table and at my elevation here in KS it is setting it to .93. SO i think this is finished and working as we wish, I think I spoke out of place and assumed the default setting at startup was .95. I think your correct that was solely a function of the autostart in an attempt to raise the startup ceiling limit.

Reply:
Try testing at KLXV (9900 feet) and VNLK (9300 ft). See https://github.com/c172p-team/c172p/issues/791#issuecomment-234784355

Reply:
We have a table now which controls the mixture to use during autostart? (I'm a bit busy with other projects and RL stuff)

Reply:
It's already been tested at the ceiling limits and works fine.
I went ahead and tested it at KLXV and that works as well.

Reply:
>The mixture is set at 0 at least in c172p-set.xml (lines 602 and 793). 

Is this what we want? Or should we set or allow it to default to 1? At 0 the knob is pulled all the way out. I always thought that looked odd to start up the simulation with that knob pulled out instead of pushed in. Unless it says different in POH?

Reply:
@wlbragg it can make sense. In the POH-180hp-552SP (at least) it says
(p. 4-17, or PDF 102/384)

> SECURING AIRPLANE
[....]
> 4. Mixture -- IDLE CUTOFF (pulled full out)
> 

Hence a likely initial state.

Reply:
Looks history (added in first commit): https://github.com/c172p-team/c172p/blame/7a2d70c69b3b7b65337cb1b08d1edc9db4bccaee/c172p-set.xml#L474

Haven't tested whether things will break if you remove it.